cmd/init.go

```diff
@@ -257,146 +274,152 @@ func (cmd *InitCmd) Run(cobraCmd *cobra.Command, args []string) {
 		}
 	}
 
+	log.WriteString("\n")
 	log.Done("Project successfully initialized")
-
-	if cmd.useCloud {
-		log.Infof("\r          \nPlease run: \n- `%s` to create a new space\n- `%s` to use an existing space\n", ansi.Color("devspace create space [NAME]", "white+b"), ansi.Color("devspace use space [NAME]", "white+b"))
-	} else {
-		log.Infof("\r          \nRun:\n- `%s` to develop application\n- `%s` to deploy application\n", ansi.Color("devspace dev", "white+b"), ansi.Color("devspace deploy", "white+b"))
-	}
+	log.Infof("\r          \nPlease run: \n- `%s` to tell DevSpace to deploy to this namespace \n- `%s` to create a new space in DevSpace Cloud\n- `%s` to use an existing space\n", ansi.Color("devspace use namespace [NAME]", "white+b"), ansi.Color("devspace create space [NAME]", "white+b"), ansi.Color("devspace use space [NAME]", "white+b"))
 }
 
-func (cmd *InitCmd) checkIfDevSpaceCloud() {
-	cmd.useCloud = true
-	connectCluster := false
+func (cmd *InitCmd) getCloudRegistryHostname() (string, error) {
+	var registryURL string
 
-	// Get provider configuration
-	providerConfig, err := cloudconfig.ParseProviderConfig()
-	if err != nil {
-		log.Fatalf("Error loading provider config: %v", err)
-	}
+	// Check if user has specified a certain provider
+	var cloudProvider *string
+	if cmd.Provider == "" {
+		// prevents EnsureLoggedIn call in GetProvider
+		registryURL = "dscr.io"
```

Comment:
This shouldn't be hardcoded since this information is retrieved from the server. This is also dependent on which provider is used